src/startup/IndexStatsModeCLIParser.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2019 DKFZ - ODCF
+ *
+ * Distributed under the MIT License (license terms are at https://github.com/dkfz-odcf/FastqIndEx/blob/master/LICENSE.txt).
+ */
+
+#ifndef FASTQINDEX_INDEXSTATSMODECLIPARSER_H
+#define FASTQINDEX_INDEXSTATSMODECLIPARSER_H
+
+#include "process/io/s3/S3Source.h"
+#include "Starter.h"
+#include "ModeCLIParser.h"
+
+class IndexStatsModeCLIParser : public ModeCLIParser {
+public:
+    IndexStatsRunner *parse(int argc, const char **argv) override {
+        auto cmdLineParser = createCommandLineParser();
+
+        auto startingReadArg = _makeUIntValueArg(
+                "s", "startingentry",
```

Comment:
```suggestion
                "s", "startingEntry",
```
Maybe `startEntry` or even better `firstEntry`?